test/mix/tasks/phx.gen.presence_test.exs

```diff
@@ -1,4 +1,4 @@
-Code.require_file "../../../installer/test/mix_helper.exs", __DIR__
+Code.require_file("../../../installer/test/mix_helper.exs", __DIR__)
```

Comment:
Those changes are unrelated, can we undo them? Same with phx.gen.presence itself.

We should mix format the whole repository at some point.